src/test/java/com/volasoftware/tinder/H2DatabaseTest.java

```diff
@@ -0,0 +1,50 @@
+package com.volasoftware.tinder;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.volasoftware.tinder.constants.Gender;
+import com.volasoftware.tinder.models.Account;
+import com.volasoftware.tinder.repositories.AccountRepository;
+import java.util.Date;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
+public class H2DatabaseTest {
```

Comment:
Because the class tests about account repository, I think more appropriate name will be AccountRepositoryTest

Reply:
class was renamed